htdocs/core/class/commonobject.class.php

```diff
@@ -8675,7 +8675,7 @@ public function isDuration($info)
 	public function isInt($info)
 	{
 		if (is_array($info)) {
-			if (isset($info['type']) && ($info['type'] == 'int' || preg_match('/^integer/i', $info['type']))) {
```

Comment:
Not sure of the fixes in `commonobject.class.php`.

Reason of theses fixes: Inserting value 0 to a MySQL column with type "smallint" was being inserted as `null`, causing the insertion to fail.